project.godot

```diff
@@ -23,7 +23,7 @@ MenuMusic="*res://scenes/menu/menu_music.tscn"
 
 [display]
 
-window/size/mode=2
+window/size/mode=4
```

Comment:
Why do you change the size mode in this pull request ?

Reply:
Hop sorry